lib/ansiblelint/utils.py

```diff
@@ -724,8 +721,7 @@ def normpath(path):
     Currently it generates a relative path but in the future we may want to
     make this user configurable.
     """
-    # convertion to string in order to allow receiving non string objects as
-    # arguments which would have failed under python2
+    # convertion to string in order to allow receiving non string objects
     return os.path.relpath(str(path))
```

Comment:
_(note)_ In the future we may want to migrate the internal paths representation to pathlib.